bls12_377/src/curves/g2_swu_iso.rs

```diff
@@ -0,0 +1,118 @@
+    );
+
+    /// COEFF_B =
+    #[rustfmt::skip]
+    const COEFF_B: Fq2 = Fq2::new(
+        MontFp!("249039961697346248294162904170316935273494032138504221215795383014884687447192317932476994472315647695087734549420"),
+        MontFp!("806998283981877041862626354975415285020485827233942100233224759047656510577433749137260740227904569833498998565"),
+    );
+
+    const GENERATOR: G2Affine = G2Affine::new_unchecked(G2_GENERATOR_X, G2_GENERATOR_Y);
+}
+
+const G2_GENERATOR_X: Fq2 = Fq2::new(G2_GENERATOR_X_C0, G2_GENERATOR_X_C1);
+const G2_GENERATOR_Y: Fq2 = Fq2::new(G2_GENERATOR_Y_C0, G2_GENERATOR_Y_C1);
+
+// sage: G2_gen
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 +
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 44471777796618567688228760095584248343372454885978087674329841655595593880133139294404651664057692271364857231527
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 :
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 191377956145194479040228903677940355038998863371661730030204479850936075480341608934735952709786495341106477933498*
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 X2
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 +
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 115206687171448860889110309021279060303629519187879257051215751573842462972180856243991157572371972099444077110343
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 :
+//(152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001*X2 1)
+// sage: G2_gen *
+// 8444461749428370424248824938781546531375899335154063827935233455917409239041
+//(0 : 1 : 0)
+/// G2_GENERATOR_X_C0 =
+pub const G2_GENERATOR_X_C0: Fq = MontFp!("44471777796618567688228760095584248343372454885978087674329841655595593880133139294404651664057692271364857231527");
+
+/// G2_GENERATOR_X_C1 =
+pub const G2_GENERATOR_X_C1: Fq = MontFp!("152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001");
+
+/// G2_GENERATOR_Y_C0 =
+pub const G2_GENERATOR_Y_C0: Fq = MontFp!("115206687171448860889110309021279060303629519187879257051215751573842462972180856243991157572371972099444077110343");
+
+/// G2_GENERATOR_Y_C1 =
+pub const G2_GENERATOR_Y_C1: Fq = MontFp!("191377956145194479040228903677940355038998863371661730030204479850936075480341608934735952709786495341106477933498");
```

Comment:
Can we directly specify this inline in `G2_GENERATOR_X` and `G2_GENERATOR_Y`?

Reply:
fixed [here](https://github.com/w3f/arkworks-curves/pull/11/commits/86ec5aae5708921f2040147d8c6fd1e30cc220e4#diff-5f6dece84d541d07eee0ae081e6e8b1c3ba876851c7591391f07bab9591122d0R74) and [here](https://github.com/w3f/arkworks-curves/pull/11/commits/86ec5aae5708921f2040147d8c6fd1e30cc220e4#diff-3658c7f4fe12cc2a0190ce2b0809d509a3bc57f5e9737869593afcc341e94f4cR68)